exercises/concept/squeaky-clean/.docs/hints.md

```diff
@@ -3,29 +3,24 @@
 ## 1. Replace any spaces encountered with underscores
 
 - [This tutorial][chars-tutorial] is useful.
-- [Reference documentation][chars-docs] for `char`s is here.
 - You can retrieve `char`s from a string using the [charAt][char-at] method.
 - You should use a [`StringBuilder`][string-builder] to build the output string.
-- See [this method][iswhitespace] for detecting spaces. Remember it is a static method.
+- Check the [Character][chars-docs] documentation for a method to detect whitespaces. Remember it is a static method.
```

Comment:
Are the other methods not static as well? We should either provide the hint everywhere or nowhere.

Reply:
Seems okay to me!